### ISSUE TITLE
[TW-660] task merge uses wrong user name

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#695]
_Timm Reitinger on 2011-11-28T22:17:25Z says:_ 

 I've set up one server (user name: timm) and two clients, merge/push/pull works fine from my ubuntu box (user name: timm), but merging from my n900 gives me a stupid error: when using 'task merge timm(at)remoteserver:/home/user/.task/', downloading remote changes works fine, but when I try to upload local changes, it uses the local username (user, i.e. user(at)remoteserver), which of course doesn't work, as the user names are different on both machines. 
This is using 1.9.4.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-11-28T22:17:25Z
Modified: 2014-02-09T02:06:32Z
```

**Comment by taskwarrior** [comment]
_Johannes Schlatow on 2011-11-28T22:23:59Z says:_ 

 Already fixed with #699.
